irc/server.go

```diff
@@ -204,6 +209,37 @@ func (s *Server) listen(addr string) {
 }
 
 //
+// websocket listen goroutine
+//
+
+func (s *Server) wslisten(addr string) {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
If we do allow multiple `wslisten`, then we might have to be careful about setting up this HTTP method handler multiple times. Maybe just pull it out of the method?